Ladybird/Android/build.gradle.kts

```diff
@@ -0,0 +1,78 @@
+import com.android.build.gradle.internal.tasks.factory.dependsOn
+
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+// FIXME: Move this somewhere nicer, with better behavior (like controlling host compiler)
+task<Exec>("buildLagomTools") {
+    commandLine = listOf("sh", "-c", "cmake -S ../../Meta/Lagom -B $buildDir/lagom-tools " +
+        " -Dpackage=LagomTools -DCMAKE_INSTALL_PREFIX=$buildDir/lagom-tools-install -GNinja" +
+        " -DSERENITY_CACHE_DIR=$buildDir/caches;" +
+        " ninja -C $buildDir/lagom-tools install")
+}
+tasks.named("preBuild").dependsOn("buildLagomTools")
+
+android {
+    namespace = "org.serenityos.ladybird"
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "org.serenityos.ladybird"
+        minSdk = 30
```

Comment:
Can we so less than 30 please? 😢

Reply:
Lagom doesn't build for API level < 30 at the moment, so I don't see a point. If we make the app buildable for less, we can lower the minimum. But I think if you try to run it on less than 30, it will fail to load some C APIs from Bionic.